lib/clients/AwsClient.py

```diff
@@ -243,7 +243,7 @@ def _create_volume(self, size, snapshot_id=None):
             kwargs = {
                 'Size': size,
                 'AvailabilityZone': self.availability_zone,
-                'VolumeType': 'standard'
+                'VolumeType': 'gp2'
```

Comment:
Is there any difference in cost for gp2 volumes? If so if some one creates the volume of gp2 type and kept attached to vm, that would increase  overcall cost fo the vm, for the bosh deployment. 
Also, this is a generic code used by all. 
So, would be better if it is made parameterised in the method.

Reply:
@sauravmndl  : agree with your comments, let the bnr lib have standard and the service which needs to use gp2 can pass this as an argument.

Reply:
@sauravmndl  We will modify the code to incorporate passing volume type as a parameter.
Reg. cost issue, will provide necessary approval.

Reply:
Agree.
Also, @ankita0811 Can you please create issue and write the summary in the issue. Also make the title of the PR smaller so as not to spill over.

Reply:
@sauravmndl  We have parametrised volume_type. Also, cost approval mail is sent..